inspire_mitmproxy/interaction.py

```diff
@@ -55,6 +56,21 @@
 logger = getLogger(__name__)
 
 
+@singledispatch
+def try_to_stringify(bytes_or_string):
+    raise Exception("Unable to stringify %r" % bytes_or_string)
+
+
+@try_to_stringify.register(str)
+def _(byte_or_string):
+    return byte_or_string
+
+
+@try_to_stringify.register(bytes)
+def _(byte_or_string):
+    return byte_or_string.decode()
```

Comment:
We might want to care about non-standard encoding here (the `{MITMRequest,MITMResponse}.original_encoding` property)

Reply:
true